packages/resources/src/Api.ts

```diff
@@ -576,13 +579,22 @@ export class Api extends Construct implements SSTConstruct {
     ///////////////////
     // Create Function
     ///////////////////
-    const lambda = Fn.fromDefinition(
-      scope,
-      `Lambda_${postfixName}`,
-      routeProps.function,
-      this.defaultFunctionProps,
-      `The "defaultFunctionProps" cannot be applied if an instance of a Function construct is passed in. Make sure to define all the routes using FunctionProps, so the Api construct can apply the "defaultFunctionProps" to them.`
-    );
+    let lambda: cdkLambda.Function | Fn;
```

Comment:
Let's duplicate the `createFunctionIntegration` function, so have `createSstFunctionIntegration` and `createLambdaFunctionIntegration`. This way, we don't have to keep checking `instanceof Fn`.

Reply:
done